web/src/layout/common/RepositoryInfo.tsx

```diff
@@ -101,7 +101,7 @@ const RepositoryInfo = (props: Props) => {
           </div>
         </div>
 
-        <div className="d-flex flex-row aling-items-baseline text-truncate">
+        <div className="d-flex flex-row align-items-baseline text-truncate">
```

Comment:
Could you please remove the style `align-items-baseline`? If we add it, the content will be misaligned